### ISSUE TITLE
ToggleGroup - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4811]
Add Accessibility information to the ToggleGroup page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the ToggleGroup component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the ToggleGroup page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance. ([PR](https://github.com/grommet/hpe-design-system/pull/4837))
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA. ([Tip issue on Grommet](https://github.com/grommet/grommet/issues/7552) - will resolve 1.4.13)

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by halocline** [comment]
Draft [PR with evaluation submitted](https://github.com/grommet/hpe-design-system/pull/4837).

TO DO: UI portion
